backend/infrahub/auth.py

```diff
@@ -170,13 +179,15 @@ async def validate_jwt_access_token(token: str) -> AccountSession:
     except Exception:
         raise AuthorizationError("Invalid token") from None
 
+    await validate_active_account(db=db, account_id=str(account_id))
```

Comment:
Unless I'm missing something, I don't think we need to validate if the account is active for each request, only when we refresh the token

Reply:
I guess it depends of the token life time. If the token life time is less than 5 minutes that's somewhat OK. If it's longer than this, it means that if someone revokes the access of an account by setting it inactive, it won't be effective until the token expires. Leading to a person still having access to the system while he should not.